docs/assets/funfacts.txt

```diff
@@ -17,11 +17,17 @@ SkyClient is for more than Skyblock, even though it started out that way.
 SkyClient is a zero profit organization; we have 40 unpaid volunteers around the globe.
 Most mod's settings are located in a command you can run when in game;\ne.g Patcher is configurable with `/patcher`.
 At one point in Minecraft, there was a rare chance of Snow Golems naturally spawning.
-When you're asked to update your mods, you can click on one to stop it\nfrom being replaced with SkyClient's latest version.
+When you're asked to update your mods, you can click on a mod's name to stop it\nfrom being replaced with SkyClient's latest version.
 Wyvest
-Forge is actually just 2 files, a jar and a json.
+Forge is actually just 2 files, a .jar and a .json.
 Make sure to drink water! You never know if you might be dehydrated :)
 People asking you to "test their mod" are always trying to rat you.\nThe only ones you can trust are ones widely trusted by the community.
 SBE bad lol
 Here's a good list of ways to stop yourself from getting ratted:\nhttps://bit.ly/HySbModSafetySCC
 Have a great day!
+Don't be scared to ask for support! SkyClient's Discord server often has over 20 concurrent support tickets open at any given day.\n\nOkay, maybe you should be mildly scared.
+#denied-mod-suggestions is an informal guideline for what mods or packs to suggest for SkyClient. Please read it!
+Feel like you're still missing a few mods? No worries-just head over to #bot-commands on the Discord server and drop the command named /modlist.
+Opal is a hydrated amorphous form of silica (SiO2·nH2O); its water content may range from 3 to 21% by weight, but is usually between 6 and 10%.
+Depending on the conditions in which it formed, opal may be transparent, translucent, or opaque, and the background color may be white, black, or nearly any color of the visual spectrum.
+If you can read this, you are definitely not on macOS. Otherwise, all you'll see is a Mojang logo.
```

Comment:
we need more people's opinions, also i feel like a lot of these would be clipped
@nacrt @UnderscoreRayless what do you think